modules/fedimint-gateway-v1/src/lib.rs

```diff
@@ -862,3 +870,52 @@ impl TryFrom<PaymentData> for SwapParameters {
         })
     }
 }
+
+#[async_trait]
+pub trait LightningV1Manager: Debug + Send + Sync {
+    // Checks if the invoice route hint last hop has source node id matching this
```

Comment:
BTW. Idiomatically in Rust in the docstrings the first line is roughly one sentence, then linebreak, then any amount of details. Newer Rust versions have clippy that is going to nag about it, so you might save some time in the future and already adapt this style.